integration-tests/integration_tests/test_scenarios/int_async_express_tests.py

```diff
@@ -65,3 +67,28 @@ def test_async_express_inbound_patient(self):
         patient_number = methods.get_section(inbound_response, 'extension', 'id', 'patient')
         self.assertEqual(patient_number, '9689177923',
                          "Async Express inbound response test failed")
+
+
+class TestSyncAsyncWrapper(TestCase):
+
+    def test_async_express_sync_async_wrap(self):
+        # the response is just an acknowledgement from spine...
+        outbound_response, _, _ = methods.get_interaction_from_template('async express',
+                                                                        'QUPC_IN160101UK05',
+                                                                        '9689177621',
+                                                                        'Asynchronous Express test',
+                                                                        sync_async=True)
+        self.assertTrue(methods.check_status_code(outbound_response, 200),
+                        "Async Express outbound test failed")
+
+        root = ET.ElementTree(ET.fromstring(outbound_response.text)).getroot()
+
+        self.assertTrue(self.does_contain_success_code(root))
+
+    def does_contain_success_code(self, root):
+        for child in root.iter():
```

Comment:
I initially tried `find` and was experimenting whilst waiting for review but it's returning no value. I _think_ it's because you have to provide the correct namespaces as a part of the search but I haven't managed to figure out the right input. I'll continue to experiment if I get time but I don't want to drain hours on this one check

Reply:
I continued to experiment with `find` and had no luck with the namespaces, but I found that using  `root.iter('urn:hl7-org:v3'queryResponseCode)` will return the desired element which I think is  a reasonable compromise